internals/cli/cmd_run.go

```diff
@@ -114,7 +113,7 @@ func runWatchdog(d *daemon.Daemon) (*time.Ticker, error) {
 			case <-wt.C:
 				// TODO: poke the API here and only report WATCHDOG=1 if it
 				//       replies with valid data.
-				systemd.SdNotify("WATCHDOG=1")
+				daemon.Notify.Send("WATCHDOG=1")
```

Comment:
I'm not really sure what `WATCHDOG=1` does, but it seems like it's pretty specific to systemd? In that case, could it not be a string in the `Send` interface, but just included in the systemd implementation? Or something like `Send(daemon.SendWatchdog)` and `Send(daemon.SendReady)` (not the best API, but spitballing here).